examples/plot_stack_masks.py

```diff
@@ -0,0 +1,131 @@
+
+# Get data and set your home working directory
+data = et.data.get_data("cold-springs-fire")
+
+###############################################################################
+# Import Example Data
+# ------------------------------
+# To get started, make sure your directory is set. Create a stack from all of the
+# Landsat .tif files (one per band) and import the ``landsat_qa`` layer which provides
+# the locations of cloudy and shadowed pixels in the scene.
+
+os.chdir(os.path.join(et.io.HOME, "earth-analytics"))
+
+# Stack the landsat bands
+# This creates a numpy array with each "layer" representing a single band
+landsat_paths_pre = glob(
+    "data/cold-springs-fire/landsat_collect/LC080340322016070701T1-SC20180214145604/crop/*band*.tif"
+)
+landsat_paths_pre.sort()
+arr_st, meta = es.stack(landsat_paths_pre)
+
+# Import the landsat qa layer
+with rio.open(
+    "data/cold-springs-fire/landsat_collect/LC080340322016070701T1-SC20180214145604/crop/LC08_L1TP_034032_20160707_20170221_01_T1_pixel_qa_crop.tif"
+) as landsat_pre_cl:
+    landsat_qa = landsat_pre_cl.read(1)
+    landsat_ext = plotting_extent(landsat_pre_cl)
+
+###############################################################################
+# Plot Histogram of Each Band in Your Data
+# ----------------------------------------
+
+# You can view a histogram for each band in your dataset by using the
+# ``hist()`` function from the ``earthpy.plot`` module.
+
+ep.hist(arr_st)
+plt.show()
+
+###############################################################################
+# Customize Histogram Plot with Titles and Colors
+# -----------------------------------------------
+
+ep.hist(
+    arr_st,
+    colors=["blue"],
+    title=[
+        "Band 1",
+        "Band 2",
+        "Band 3",
+        "Band 4",
+        "Band 5",
+        "Band 6",
+        "Band 7",
+    ],
+)
+plt.show()
+
+###############################################################################
+# View Single Band Plots
+# -----------------------------------------------
+# Next, have a look at the data, it looks like there is a large cloud that you
+# may want to mask out.
+
+ep.plot_bands(arr_st)
+plt.show()
+
+
+###############################################################################
+# Mask the Data
+# -----------------------------------------------
+
+# You can use the earthpy ``mask()`` function to handle this cloud.
+# To begin you need to have a layer that defines the pixels that
+# you wish to mask. In this case, the ``landsat_qa`` layer will be used.
+
+ep.plot_bands(
+    landsat_qa,
+    title="The Landsat QA Layer Comes with Landsat Data\n It can be used to remove clouds and shadows",
+)
+plt.show()
+
+
+###############################################################################
+# Plot The Masked Data
+# ~~~~~~~~~~~~~~~~~~~~~
+# Now apply the mask and plot the masked data. The mask applies to every band in your data.
+
+# Generate array of all possible cloud / shadow values
```

Comment:
There are a lot of magic numbers in this code chunk - maybe you can point users to something that explains where these come from.

Reply:
i added a few sentences at the top about the data. we could link to the EDS lesson on this which is probably the best option in the long term but for now i just added some text.